description_templates/object-storage-storage:cloudscale.gotmpl

```diff
@@ -1,5 +1,5 @@
-{{- template "_s3_storage" . -}}
-{{- define "_s3_storage" -}}
+{{- template "_s3_storage_cloudscale" . -}}
+{{- define "_s3_storage_cloudscale" -}}
```

Comment:
why are the golden files for cloudscale empty?

Reply:
They are not empty, they were recreated after `make gen-golden`